jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/core/assembly/DockerAssemblyManager.java

```diff
@@ -391,6 +357,8 @@ private void processJKubeProjectAssemblyFile(
         final String destinationFilename = Optional.ofNullable(assemblyFile.getDestName()).orElse(sourceFile.getName());
         final File destinationFile = new File(outputDirectory, destinationFilename);
         FileUtil.copy(sourceFile, destinationFile);
+        // TODO: Need to add permission (Probably change AssemblyFile model to have this)
```

Comment:
Should we file a separate issue for this pending task? Wdyt?

Reply:
See PR description in point "No new bugs, code smells...". Everything already linked up (I don't really like leaving ToDos in code, but this one was already quite big without further changes, and I already left the placeholder for the future fix)

Reply:
Ah, sorry. Looks like I didn't see linked issue in description